File: src/main/java/com/pengrad/telegrambot/model/request/InlineKeyboardMarkup.java
package com.pengrad.telegrambot.model.request;

import com.pengrad.telegrambot.model.request.Keyboard;

public class InlineKeyboardMarkup extends Keyboard {

	private InlineKeyboardButton[][] inline_keyboard;

	public InlineKeyboardMarkup( InlineKeyboardButton[] ... keyboards ) {

		this.inline_keyboard = keyboards;
	}

	public InlineKeyboardButton[][] getInline_keyboard() {

		return inline_keyboard;
	}

	public void setInline_keyboard(
			InlineKeyboardButton[][] inline_keyboard ) {

		this.inline_keyboard = inline_keyboard;
	}

}

File: core/src/commonMain/kotlin/com/pengrad/telegrambot/model/request/InlineKeyboardMarkup.kt
package com.pengrad.telegrambot.model.request

import java.io.Serializable

/**
 * stas
 * 8/4/15.
 */
class InlineKeyboardMarkup(vararg keyboard: Array<InlineKeyboardButton>) : Keyboard(), Serializable {

    private val inline_keyboard: Array<out Array<InlineKeyboardButton>> = keyboard

    companion object {
        private const val serialVersionUID = 0L
    }
}
